client/my-sites/checkout/checkout/free-cart-payment-box.jsx

```diff
@@ -81,6 +83,20 @@ class FreeCartPaymentBox extends React.Component {
 		return '';
 	};
 
+	getDomainCreditIllustration = () => {
+		const cart = this.props.cart;
+
+		if ( ! cart.has_bundle_credit ) {
+			return;
+		}
+
+		return (
+			<span className="checkout__free-domain-credit-illustration">
+				<img src="/calypso/images/illustrations/custom-domain.svg" alt="" />
```

Comment:
Should we add an `alt` tag here or just get rid of `alt` entirely?

Reply:
The pre-commit checks flagged it as necessary, but since it's decorative it says we can leave it empty.